options.css

```diff
@@ -10,9 +10,7 @@
 }
 
 body {
-  /* background: linear-gradient(hsl(271, 77%, 15%), #27012b); */
-  background: #242a36;
```

Comment:
The current background looks great but the background file is around 16 KB and would unnecessarily increase the extension size. It has almost tripled already since v1. So can you please revert it back to the previous one.

Reply:
I have made some changes - Reduced the file sizes of ( bg.png and bin.png) to almost 1KB. 
and this is how it looks now.

![image](https://user-images.githubusercontent.com/43717403/68045755-4884c980-fd00-11e9-9576-a7c3fa844d9f.png)

but if you still want to revert this to just plain color, do tell me. :)